daemon/container.go

```diff
@@ -1009,27 +997,38 @@ func (container *Container) Copy(resource string) (io.ReadCloser, error) {
 	if err := container.Mount(); err != nil {
 		return nil, err
 	}
+	var paths []string
+	unmount := func() {
+		for _, p := range paths {
+			syscall.Unmount(p, 0)
```

Comment:
It might not be relevant, but why not use `mount.Unmount`, if only for the sake of symmetry with the `mount.Mount` call below?